src/pages/register.tsx

```diff
@@ -0,0 +1,130 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+const Register = () => {
+  const { t, i18n } = useTranslation();
+  const [currentLanguage, setCurrentLanguage] = useState('en');
+  const authenticationType = useAuthenticationTypes();
+
+  const items: MenuProps['items'] = [
+    {
+      label: 'English',
+      key: 'en',
+      onClick: () => {
+        setCurrentLanguage('en');
+        i18n.changeLanguage('en');
+      },
+    },
+    {
+      label: 'Tiếng Việt',
+      key: 'vi',
+      onClick: () => {
+        setCurrentLanguage('vi');
+        i18n.changeLanguage('vi');
+      },
+    },
+  ];
+
+  return (
+    <div className="flex flex-row h-screen w-screen justify-evenly p-4 md:p-12 bg-[#F6F6F6]">
+      <div className="relative md:flex md:items-center md:justify-center hidden">
+        <Image
+          className="relative lg:max-w-[700px] md:max-w-[90%]"
+          src={MockupIC}
+          preview={false}
+        />
+        <div className="hidden md:block absolute top-6 left-8 text-3xl font-medium">
+          {t('common.text.signUpToPayment')}
+        </div>
+        <div className="hidden md:block absolute top-[90px] left-8 text-sm font-light text-[#4F555A] max-w-[300px]">
+          {t('common.text.account')}{' '}
+          <Link to="/sign-in" className="text-[#56B280] font-semibold underline">
+            {t('common.text.signInHere')}
+          </Link>
+        </div>
+      </div>
+      <div className="flex flex-col">
+        <div className="flex items-center justify-end w-full gap-6 p-4">
+          <Dropdown menu={{ items }} trigger={['click']}>
+            <Space>
+              {currentLanguage === 'en' ? 'English' : 'Tiếng Việt'}
+              <DownOutlined />
+            </Space>
+          </Dropdown>
+          {authenticationType.map((item) => (
+            <Link
+              key={item.value}
+              to={item.path}
+              className={`text-sm text-[#56B280] font-semibold px-4 py-2 bg-white shadow-lg rounded-2xl ${
+                location.pathname === item.path ? 'border-b-2 border-[#56B280]' : ''
+              }`}
+            >
+              {' '}
+              {item.text}
+            </Link>
+          ))}
+          <Button className="bg-[#56B280]  px-4 py-2" type="primary">
+            <Link to="/home">{t('common.button.home')}</Link>
+          </Button>
+        </div>
+
+        <div className="flex flex-col gap-3 flex-grow justify-center">
+          <div className="md:hidden text-center mb-4 text-3xl font-medium">
+            {' '}
+            {t('common.text.signUpToPayment')}{' '}
+          </div>
+          <div className="md:hidden text-center mb-4 text-sm font-light text-[#4F555A]">
+            {t('common.text.account')}{' '}
+            <Link to="/sign-in" className=" text-[#56B280] font-semibold underline">
+              {t('common.text.signInHere')}
+            </Link>
+          </div>
+
+          <Input
```

Comment:
Can you try create component Input with css, props like value, text, css ..., and you call inside. it make you don't using css many time for same code